relayer/src/chain/handle/requests.rs

```diff
@@ -1,7 +1,8 @@
 //! Domain-type definitions for requests that a
-//! [`ChainHandle`] can accept. These types trickle
-//! down to the [`ChainEndpoint`] trait. Any chain
-//! that the ibc-relayer library supports will have
+//! [`ChainHandle`](crate::chain::handle::ChainHandle) can accept.
+//! These types trickle down to the
+//! [`ChainEndpoint`](crate::chain::ChainEndpoint) trait.
```

Comment:
Good, but stylistically footnote-style markup reads better in code, like in my suggestion above.